package.json

```diff
@@ -37,29 +37,30 @@
     "win32"
   ],
   "dependencies": {
-    "@yaireo/tagify": "^3.6.2",
```

Comment:
Why all the package downgrades?

Reply:
My bad, I had a stale copy of package.json. Fixed.